src/libraries/System.Net.Http/src/System/Net/Http/HttpMethod.cs

```diff
@@ -66,7 +66,8 @@ public static HttpMethod Patch
 
         // Don't expose CONNECT as static property, since it's used by the transport to connect to a proxy.
         // CONNECT is not used by users directly.
-
+        /// <summary>Represents an HTTP CONNECT protocol method.</summary>
+        /// <value>Returns <see cref="T:System.Net.Http.HttpMethod" />.</value>
```

Comment:
Here in triple slash, you don't have to add the cref with the prefix and the full name. Just mention it as you would in source code.

Did you see any build failures?
Assuming the right namespace is included among the using directives, this should be:

```suggestion
        /// <value>Returns <see cref="HttpMethod" />.</value>
```

But since it's the value, it should be expanded a bit more to describe what the Http Method represents.

From: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#value
> The <value> tag lets you describe the value that a property represents.

Reply:
Would something like "Returns <see cref="HttpMethod" /> instance that represents the HTTP CONNECT method." be okay? If so I will fix it for other members in the class as well